src/Tempest/Database/src/IsDatabaseModel.php

```diff
@@ -76,6 +76,11 @@ public static function query(): ModelQueryBuilder
         return new ModelQueryBuilder(self::class);
     }
 
+    public static function getModelInstanceClass(mixed ...$params): string
```

Comment:
Perhaps in this case it makes more sense to pass in an array instead of spreading the arguments. Tempest uses argument spreading when the user needs control over the names of the variables, which in this situation isn't really the case.

Reply:
Done in latest commit